cmd/set.go

```diff
@@ -69,22 +87,24 @@ Sets the value of 'endpoint' equal to 'east-db' in the 'common' config (the defa
 $ fab set --subcomponent "myapp.mysubcomponent" data.replicas=5 
 
 Sets the subkey "replicas" in the key 'data' equal to 5 in the 'common' config (the default) for the subcomponent 'mysubcomponent' of the subcomponent 'myapp'.
+
+$ fab set --subcomponent "myapp.mysubcomponent" data.replicas=5 --no-new-config-keys
+
+Uses the --no-new-config-keys switch to prevent the creation of new config .
```

Comment:
Can we change 'Uses' -> 'Use' and get rid of the space after the period so that this reads smoother?